cloudify_awssdk/common/decorators.py

```diff
@@ -251,6 +251,11 @@ def wrapper_inner(**kwargs):
                 raise OperationRetry(
                     '%s ID# "%s" is still in a pending state.'
                     % (resource_type, iface.resource_id))
+
+            elif status_good and status in status_good:
```

Comment:
Why?

Reply:
Simply because any resource depend on waiting status like `vpc`, `instances` we are saving the `create_response` object which will be held the old state of the object so this check will make sure that we have the latest state of the response returned for the new resource 

check for example the vpc https://github.com/cloudify-incubator/cloudify-awssdk-plugin/blob/CY-865-remove-references-to-old-aws-plugin-from-example/cloudify_awssdk/ec2/resources/vpc.py#L110